addons/point_of_sale/static/src/xml/pos.xml

```diff
@@ -900,7 +900,7 @@
             <line t-foreach='payments' t-as='payment'>
                 <left><t t-esc='payment.name' /></left>
                 <right>
-                    <t t-esc='payment.total'/>
```

Comment:
there is no precision here?

Reply:
I prefered to format all potential floats ....

Reply:
No I mean, we do not specify any precision parameter as we do below

Reply:
ha, sorry.
in JS, the widget itself fetches the currency to get the precision
https://github.com/odoo/odoo/blob/10.0/addons/point_of_sale/static/src/js/widget_base.js#L37

in python, the qweb widget doesn't:
https://github.com/odoo/odoo/blob/10.0/odoo/addons/base/ir/ir_qweb/fields.py#L136